packages/DataGrid/src/components/Cell/Cell.js

```diff
@@ -21,36 +21,13 @@ const propTypes = {
   hasZebraStripes: PropTypes.bool.isRequired,
 };
 
-const Cell = React.forwardRef((props, ref) => {
+export default function Cell(props) {
   const { style, gridId, columnIndex, rowIndex, column, data, a11yText, hasZebraStripes } = props;
-  const [isActiveCell, setIsActiveCell] = React.useState(false);
-  const [isActiveRow, setIsRowActive] = React.useState(false);
-
-  React.useImperativeHandle(ref, () => ({
```

Comment:
adding this imperative API for the cell so would be possible to do row hover or mouseover/mouseout event on the table turn to be heavy, so I decided to remove it, I'm not aware of any DataGrid using these features, so as I commented previously I think a better option for this thing of interaction is to look into [recoiljs](https://recoiljs.org/).